app/Http/Controllers/CompetitionsController.php

```diff
@@ -7,7 +7,7 @@
 
 class CompetitionsController extends Controller
 {
-    private $validationRules = [
+    public static $validationRules = [
```

Comment:
I'm obviously new to this so I'm not sure why its bad practice, especially since the form is a partial. If we're making forms partials then I'm assuming it's built to be used in more than one place? 

If we did what you proposed there, Chris/Hillary could be going back and editing multiple competitions potentially if they didn't like the default dates. I feel like saving them a few minutes of time might be worth the small compromise we're making here.

A possible alternative? Can we make a function for creating/saving competitions, that both the competition controller & waiting room controller use?